rhdm70-image-streams.yaml

```diff
@@ -0,0 +1,41 @@
+kind: List
+apiVersion: v1
+metadata:
+  name: rhdm70-image-streams
+  annotations:
+    description: ImageStream definitions for Red Hat Decision Manager 7.0
+items:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: rhdm70-decisioncentral-openshift
+    annotations:
+      openshift.io/image.insecureRepository: 'true'
+      openshift.io/display-name: Red Hat Decision Manager Central 7.0
+  spec:
+    dockerImageRepository: docker-registry.engineering.redhat.com/rhdm-7/rhdm70-decisioncentral-openshift
```

Comment:
We should use the same pattern than the existing jboss-image-streams: https://github.com/jboss-openshift/application-templates/blob/master/jboss-image-streams.json#L40

Reply:
Sorry, I am not sure I understand. `docker-registry.engineering.redhat.com` is the only location (I am aware of) where the images are at this point. For releases we we would of course need to change the URL.

Reply:
I think that we shouldn't put this file into public repo until there are available images in registry.access.redhat.com.

Reply:
I can of course delete the file.

Reply:
I meant, you should change the docker repository like this:
                        "from": {
                            "kind": "DockerImage",
                            "name": "registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:1.3"
                        }

Reply:
Ah, ok. Will change that.